.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You will need to revert changes to this file.

Reply:
Removed the files accidentally added to the git